packages/prometheus-operator/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@kubernetes-models/prometheus-operator",
+  "version": "0.1.4",
+  "description": "Kubernetes prometheus-operator models",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tommy351/kubernetes-models-ts.git"
+  },
+  "homepage": "https://github.com/tommy351/kubernetes-models-ts",
+  "author": "Julien Bouquillon <julien@bouquillon.com>",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "build:alertmanagers": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml --output ./gen",
+    "build:podmonitors": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml --output ./gen",
+    "build:probes": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_probes.yaml --output ./gen",
+    "build:prometheuses": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml --output ./gen",
+    "build:prometheusrules": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml --output ./gen",
+    "build:servicemonitors": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml --output ./gen",
+    "build:thanosrulers": "crd-generate --input https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml --output ./gen",
+    "build": "npm-run-all build:*",
+    "prepack": "cp package.json README.md dist/",
+    "clean": "rimraf gen"
+  },
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist"
+  },
+  "keywords": [
+    "kubernetes"
+  ],
+  "dependencies": {
+    "@kubernetes-models/base": "^0.1.4",
+    "@kubernetes-models/validate": "^0.1.2",
+    "kubernetes-models": "^0.8.1",
```

Bump kubernetes-models to `1.0.0`.

```suggestion
    "@kubernetes-models/base": "^1.0.0",
    "@kubernetes-models/validate": "^1.0.0",
    "kubernetes-models": "^1.0.0",
```